merge_operator.go

```diff
@@ -78,8 +79,8 @@ type MultiMerger interface {
 }
 
 // NewNativeMergeOperator creates a MergeOperator object.
-func NewNativeMergeOperator(c *C.rocksdb_mergeoperator_t) MergeOperator {
-	return &nativeMergeOperator{c}
+func NewNativeMergeOperator(c unsafe.Pointer) MergeOperator {
```

Comment:
Same as above

Reply:
We are exporting NewNative funcs for SliceTransform, CompactionFilter, MergeOperator